__tests__/app.test.js

```diff
@@ -66,6 +67,7 @@ describe('GET /api/articles/:article_id', () => {
                 expect(typeof article.created_at).toBe('string')
                 expect(typeof article.votes).toBe('number')
                 expect(typeof article.article_img_url).toBe('string')
+                expect(typeof article.comment_count).toBe('number')
```

Comment:
We should never alter previous tests when it's at all possible to make a new test when we add more functionality.
Leaving previous tests untouched let's us follow TDD practise of making sure all previous functionality is left intact when we refactor 🧪 
The ideal approach here would be a separate new test in this describe block that just checks for the `comment_count` property 🔍